practice-2/count_word_book.py

```diff
@@ -0,0 +1,82 @@
+"""
+This module contains a code for exercises 13-2 related to:
+Think Python, 2nd Edition
+Chapter 13: Case Study: Data Structure Selection
+
+Modify the program to count the total number of words in the book, and the
+number of times each word is used. Print the number of different words used in the book.
+
+Download Ebook.txt file
+"""
+
+import string
+
+def clean_word():
+    """
+    Read a file and count its word.
+
+    returns: list words: list and count: int
```

Comment:
Invalid docstring, and with a return statement that returns multiple values, you can use it in the following way:
```
Returns:
    - cleaned_words: list
    - count: int

Reply:
Please check for the remaining files.